### DISCUSSION TITLE
Code changes

### DISCUSSION THREAD

**Discussion by oytunturk** [NVIDIA-NeMo/NeMo#3763] [Q&A] [answered]
My code changes don't seem to have any effect even after running `pip install nemo_toolkit['all']`. Should I run another pip command? Thanks!

**Comment by titu1994** [comment]
Could you elaborate on what is meant by "my code changes don't take effect"?

**Comment by oytunturk** [comment]
Changing anything in nemo, let’s say in one of the model architectures or
config file structure.   It seems to neglect any changes and still runs the
non-modified version. I think I’m missing some kind of a
reinstallation/recompilation step.

**Comment by MaximumEntropy** [comment, marked as answer]
You probably want to do `pip install -e ...` to have your code changes take effect.

**Reply by oytunturk** [reply]
Thanks! Running `pip install -e .` from nemo/NeMo/ folder worked nicely.
